src/cascade_at/jobmon/workflow.py

```diff
@@ -1,51 +1,51 @@
 import os
-import getpass
 from typing import Optional
 
-from cascade_at.core.db import swarm
+from cascade_at.cascade.cascade_operations import _CascadeOperation
+from cascade_at.core.db import client
 from cascade_at.core.log import get_loggers
 
 LOG = get_loggers(__name__)
 
-Workflow = swarm.workflow.workflow.Workflow
-BashTask = swarm.workflow.bash_task.BashTask
-ExecutorParameters = swarm.executors.base.ExecutorParameters
+ExecutorParameters = client.api.ExecutorParameters
+Tool = client.api.Tool
+Task = client.task.Task
+Template = client.task_template.TaskTemplate
 
 
-class COBashTask(BashTask):
-    """
-    Just a little modification to BashTask so that it has
-    an attribute for upstream commands in order for us to index
-    the cascade operations correctly.
-    """
-    def __init__(self, upstream_commands=None, **kwargs):
-        super().__init__(**kwargs)
-        if upstream_commands is None:
-            upstream_commands = []
-        self.upstream_commands = upstream_commands
-        LOG.info(f"Created task with command {self.command} with "
-                 f"{len(self.upstream_commands)} upstream commands.")
+class JobmonConstants:
+    EXECUTOR = "SGEExecutor"
```

Comment:
You will also want to import whichever executor(s) you are going to use otherwise I think you will run into an error. For SGEExecutor `from jobmon.client.execution.strategies.sge import SGEExecutor`